README.md

```diff
@@ -5,15 +5,14 @@
 1. Select latest release (tag) that is less than or equal to a version of the cluster you plan to use CLI with.
 2. Download image from docker repository `docker pull vastdataorg/vast-db-cli:4.7.0-sp3`
 3. Download `vast-db-cli` from this repository.
-4. Move `vast-db-cli` into the `/bin/` directory.
-5. Make `vast-db-cli` executable: `chmod +x /bin/vast-db-cli`
-6. Execute the script.
+4. Make `vast-db-cli` executable: `chmod +x vast-db-cli`
+5. Configure it: `./vast-db-cli configure`
 
 The Vast DB CLI is now installed.
 
 ### Getting Help
 For information about how to use the Vast DB CLI, run the following command:
 
 ```shell
-$ vast-db-cli --help
+$ ./vast-db-cli --help
 ```
```

Comment:
Moving to `/bin` is needed  so that we have a pretty execution cmd (without the need of `./`).

While moving to `/bin/` worked fine on CentOS systems, you're right, that on latest MacOS there are issues:
```
▶ sudo mv ~/Downloads/vast-db-cli /bin/
mv: fastcopy: open() failed (to): /bin/vast-db-cli: Operation not permitted
```

Still, I don't think we should encourage to use `./vast-db-cli`. Instead, I suggest to add "Move `vast-db-cli` into the `/usr/local/bin/` directory."

It should do the trick.

Reply:
It's pretty unexpected that a user is downloading a script from the internet, making it executable, and then globally available to all users (either in `/bin` or in `/usr/local/bin`. It's much more common for a user to just run it locally. I do not think we should be telling users to do this.